### PR TITLE
[#172992652] Activate bonus screen

### PR DESCRIPTION
**Pull request by fabriziofff**
This pr depends on (#1846).

**Short description:**
This pr adds the screen that allow to see:
- The amount of bonus available
- A Description about the bonus
- A resume of the family members

And allows to:
- Activate the bonus (request the identification)

Screen            |  Activate the bonus Action
:-------------------------:|:-------------------------:
|  

**List of changes proposed in this pull request:**
- `ts/features/bonusVacanze/screens/ActivateBonus/ActivateBonusComponent.tsx` a component that implements all the graphic required by the screen.
- `ts/features/bonusVacanze/screens/ActivateBonus/ActivateBonusScreen.tsx` that uses the `ActivateBonusComponent` and tie it with the business logic.

### CODE REVIEW

File: ts/features/bonusVacanze/screens/ActivateBonus/ActivateBonusScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  // TODO: link with the right dispatch action
+  onCancel: () => {
+    RTron.log("CANCEL");
+  },
+  // When the user choose to activate the bonus, verify the identification
+  onActivateBonus: () => requestIdentification(dispatch)
+});
+
+// TODO: remove this mock data after the linking with redux
+const mockFamiluMembers: ReadonlyArray<FamilyMember> = [
```
Comment (Undermaken) [inline ts/features/bonusVacanze/screens/ActivateBonus/ActivateBonusScreen.tsx:74]: To keep aligned with official types (I hope) and use mocked data only from a single source, you could use this https://github.com/pagopa/io-app/blob/d92fd5cdf05762c17a225d2c8e24db1e3f9b5d88/ts/features/bonusVacanze/mock/mockData.ts#L41 (Just added)

File: ts/features/bonusVacanze/screens/ActivateBonus/ActivateBonusScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  {
+    name: "Piero" as NonEmptyString,
+    surname: "Filippi" as NonEmptyString,
+    fiscal_code: "CHDJFM74KL244K" as FiscalCode
+  }
+];
+
+const mapStateToProps = (_: GlobalState) => ({
+  // TODO: link with the right reducer
+  bonusAmount: 500,
```
Comment (Undermaken) [inline ts/features/bonusVacanze/screens/ActivateBonus/ActivateBonusScreen.tsx:94]: As for family members you could use a bonus mocked data from here https://github.com/pagopa/io-app/blob/d92fd5cdf05762c17a225d2c8e24db1e3f9b5d88/ts/features/bonusVacanze/mock/mockData.ts#L59
(we should display tax_benefit too)

File: ts/features/bonusVacanze/screens/ActivateBonus/ActivateBonusComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+/**
+ * This component displays generic information about the bonus, including the {@link Props.bonusAmount} and
+ * the {@link Props.familyMembers}.
+ * @param props
+ * @constructor
+ */
+export const ActivateBonusComponent: React.FunctionComponent<Props> = props => {
+  const description = I18n.t("bonus.bonusVacanza.activateBonus.description", {
+    amount: props.bonusAmount
```
Comment (Undermaken) [inline ts/features/bonusVacanze/screens/ActivateBonus/ActivateBonusComponent.tsx:70]:
```suggestion
    amount: formatNumberCentsToAmount(props.bonusAmount, true)
```

File: ts/features/bonusVacanze/screens/ActivateBonus/ActivateBonusScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  onCancel: () => {
+    RTron.log("CANCEL");
+  },
+  // When the user choose to activate the bonus, verify the identification
+  onActivateBonus: () => requestIdentification(dispatch)
+});
+
+const mapStateToProps = (_: GlobalState) => ({
+  // TODO: link with the right reducer
+  bonusVacanze: mockedBonus,
```
Comment (fabriziofff) [inline ts/features/bonusVacanze/screens/ActivateBonus/ActivateBonusScreen.tsx:75]: @Undermaken Are we sure to use this data model here? In this phase we should have only the response from `/eligibility` with the payload with values `family_members`, `max_amount` and `max_tax_benefit`. Seems to me that the model `BonusVacanzaMock` represents an already request bonus.

Comment (Undermaken) [reply ts/features/bonusVacanze/screens/ActivateBonus/ActivateBonusScreen.tsx:75]: @fabriziofff Yes, you are totally right.
I lost the point where we are.
At this point we should have something like this

```json
{
  "family_members": [
    {
      "name": "Mario",
      "surname": "Rossi",
      "fiscal_code": "EFCMZZ80A12L720R"
    },
    {
      "name": "Giulia",
      "surname": "Rossi",
      "fiscal_code": "CDCMQQ81A12L721R"
    },
    {
      "name": "Piero",
      "surname": "Rossi",
      "fiscal_code": "ABCMYY82A12L722R"
    }
  ],
  "max_amount": 50000,
  "max_tax_benefit": 3000,
  "id": "76d02f6d-ca02-4b1c-8b6a-4b8ca71d6293",
  "status": "ELIGIBLE"
}
```
Please could you revert?

**Pull request merged**